packages/core/src/wallet/metamask_snap/signer.ts

```diff
@@ -0,0 +1,40 @@
+import {
```

Comment:
For now we have a dummy signer to satisfy the types, a new issue should be made to take in the injected provider and complete the interface.